docs/goals.md

```diff
@@ -18,7 +18,7 @@ Goals
 
    No matter what you have installed on your system, you should be able to run
    Requests. We use cURL if it's available, and fallback to sockets otherwise.
-   We require only a baseline of PHP 5.6, leaving the choice of PHP minimum
+   We require only a baseline of PHP 5.6.20, leaving the choice of PHP minimum
    requirement fully in your hands, and giving you the ability to support many
    more hosts.
```

Comment:
Good for this change now, but I think this sentence needs rephrasing, regarding the "baseline", which doesn't mean much, and we're actually NOT "leaving the choice of PHP minimum requirement fully in your hands"...

Reply:
IIRC this is part of the "in comparison to other HTTP frameworks", highlighting the _low_ PHP version Requests still supports in contrast to other HTTP frameworks, which raise the minimum PHP version far more frequently and aggressively.